code/script.js

```diff
@@ -1,213 +1,218 @@
 // All the DOM selectors stored as short variables
-const board = document.getElementById('board')
-const questions = document.getElementById('questions')
-const restartButton = document.getElementById('restart')
+const board = document.getElementById("board");
+const questions = document.getElementById("questions");
+const restartButton = document.getElementById("restart");
+const filter = document.getElementById("filter");
+const playAgainButton = document.getElementById("playAgain");
+const winSound = document.getElementById("win");
+const loseSound = document.getElementById("lose");
+const flipSound = document.getElementById("flip");
 
 // Array with all the characters, as objects
 const CHARACTERS = [
   {
-    name: 'Jabala',
-    img: 'images/jabala.svg',
-    hair: 'hidden',
-    eyes: 'hidden',
-    accessories: ['glasses', 'hat'],
-    other: []
+    name: "Jabala",
```

Comment:
How come you changed all of the ' to " ?  Did you do that manually? Alot of work!

Reply:
haha I did not change them?! maybe some of my extensions did? lite the "prettier" or something? I def would not put myself through that 😄